qiskit/algorithms/gradients/__init__.py

```diff
@@ -17,74 +17,94 @@
 
 .. currentmodule:: qiskit.algorithms.gradients
 
-
-Estimator Gradients
-===================
+Base Classes
+============
 
 .. autosummary::
    :toctree: ../stubs/
 
    BaseEstimatorGradient
-   DerivativeType
-   FiniteDiffEstimatorGradient
-   LinCombEstimatorGradient
-   ParamShiftEstimatorGradient
-   SPSAEstimatorGradient
-   ReverseEstimatorGradient
+   BaseQGT
+   BaseSamplerGradient
+   EstimatorGradientResult
+   SamplerGradientResult
+   QGTResult
 
-Sampler Gradients
-=================
+Finite Differences
+==================
 
 .. autosummary::
    :toctree: ../stubs/
 
-   BaseSamplerGradient
+   FiniteDiffEstimatorGradient
    FiniteDiffSamplerGradient
+
+Linear Combination of Unitaries
+===============================
+
+.. autosummary::
+   :toctree: ../stubs/
+
+   LinCombEstimatorGradient
    LinCombSamplerGradient
+   LinCombQGT
+
+Parameter Shift Rules
+=====================
+
+.. autosummary::
+   :toctree: ../stubs/
+
+   ParamShiftEstimatorGradient
    ParamShiftSamplerGradient
-   SPSASamplerGradient
 
-Quantum Geometric Tensor
-========================
+Quantum Fisher Information
+==========================
+
 .. autosummary::
    :toctree: ../stubs/
 
-   BaseQGT
-   LinCombQGT
+   QFIResult
    QFI
-   ReverseQGT
 
-Results
+Reverse
 =======
```

Comment:
I'm not sure how well known "reverse" is in literature, since this is a classical calculation maybe this would be more suitable?
```suggestion
Classical Methods
=================
```